packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/propertyComponent.css

```diff
@@ -12,3 +12,48 @@
   display: flex;
   align-items: center;
 }
+
+.property-display-toggle {
+  position: relative;
+}
+
+.property-display-toggle .tooltiptext {
```

Comment:
Is it possible to use this component for the tooltip? https://cloudscape.design/components/popover/?tabId=playground

Reply:
If this isn't possible, can we make some component for tooltips? I know we also have them in this component - `packages/dashboard/src/components/palette/icons/index.tsx`

Reply:
no, popover component is not working as the expected. needs to open on mouse over